scripts/opposed-wfrp4e.js

```diff
@@ -110,6 +113,153 @@ class OpposedWFRP
     this.evaluateOpposedTest(this.attacker, this.defender);
   }
 
+  /*Known Bugs: attempting to reroll causes it to not reroll at all, actually. Manually editing cards causes a duplicate result card at the end.
+  *
+  *
+  *
+  */
+  static async checkPostModifiers(attacker, defender){
+    console.log(attacker)
+    console.log(defender)
+
+    let attackerMessage = game.messages.get(attacker.messageId)
+    let defenderMessage = game.messages.get(defender.messageId)
+
+    if(attackerMessage == null || attackerMessage.data.flags.data.hasBeenCalculated || defenderMessage.data.flags.data.hasBeenCalculated)
+      return false;
+
+    let didModifyAttacker, didModifyDefender = false;
+
+    let modifiers = {
+      attackerTarget: 0,
+      defenderTarget: 0,
+      attackerSL: 0,
+      defenderSL: 0,
+      message: [],
+    }
+
+    // Things to Check:
+    // Weapon Length DONE
+    // Fast Weapon Property DONE
+    // Size 
+    // Done - Weapon Defending: You suﬀer a penalty of –2 SL for each step larger your opponent is when using Melee to defend an Opposed Test
+    // Done - To Hit Modifiers: +10 Bonus if smaller
+    // Done - Ranged to Hit Modifiers : You gain a hefty bonus when shooting at larger targets (Ex. +40 to hit Enormous).
+    //Shooting at smaller targets?
```

Comment:
This rule is pretty dumb, but if you succeed because of the positive bonuses from Size, you succeed with +0 maximum.

Reply:
That's uh, pretty weird. Not really sure how to implement that, but I'll take a look later.

Reply:
Moved to issue: https://github.com/CatoThe1stElder/WFRP-4th-Edition-FoundryVTT/issues/468